hyprctl/main.cpp

```diff
@@ -419,6 +427,5 @@ int main(int argc, char** argv) {
         exitStatus = request(fullRequest);
     }
 
-    std::cout << std::endl;
```

Comment:
this is useful as it flushes stdout, though you can also just flush

Reply:
Why is flushing useful?

Reply:
to avoid half-written logs